src/lib/protocols/stun.c

```diff
@@ -332,8 +334,8 @@ static ndpi_int_stun_t ndpi_int_check_stun(struct ndpi_detection_module_struct *
         case 0x4002:
           /* These are the only messages apparently whatsapp voice can use */
           *app_proto = NDPI_PROTOCOL_WHATSAPP_CALL;
-          return(NDPI_IS_STUN);
-
+          //return(NDPI_IS_STUN); //necessary because the allocate success response packets (from the PCAP file) don't come through TODO: figure out why
+          break;
```

Comment:
This is likely an unrelated issue that we can address later: right now we end the dissection when we have a valid sub-proto (whatsapp Call in this case); we might need to keep parsing more pkts/attributes looking for more metadata.

Anyway, for the time being, to test your code simply use a pcap with only one Allocate Response pkt